### PR TITLE
Adds Dapr Bot & Fossa Dependency License Scan

### LINKED ISSUE #359: Fixing dependencies with bad license
Main issue: https://github.com/dapr/dapr/issues/4236

Please refer to the parent issue to help with fixing existing issues with bad licenses. 

This is required for Dapr to be compliant with CNCF guidelines, so please add this issue to the SDK planning accordingly.

### PR DESCRIPTION
**Pull request by berndverst**
# Description

Fixes #359

Enables license scan for dependencies going forward

### CODE REVIEW

**Review by artursouza — APPROVED** [review]

**Review by artursouza — CHANGES_REQUESTED** [review]

**Comment by berndverst** [comment]
Adds `/ok-to-test` for FOSSA scanning
Adds `/assign`
Adds `/make-me-laugh` 😆 

Also marks issues as stale after 60 days.

File: .github/workflows/fossa.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - name: "Run FOSSA Scan"
+        uses: fossas/fossa-action@main # Use a specific version if locking is preferred
+        with:
+          api-key: ${{secrets.FOSSA_API_KEY}}
+
+      - name: "Run FOSSA Test"
+        uses: fossas/fossa-action@main # Use a specific version if locking is preferred
+        with:
+          api-key: ${{secrets.FOSSA_API_KEY}}
+          run-tests: true
```
Comment (artursouza) [inline .github/workflows/fossa.yaml:42]: You need to comment back in the PR with the result, similar to how is done here: https://github.com/dapr/dapr/blob/3ab31bf4762df47a5c8a32245ec6a9f3c5c3a50e/.github/workflows/dapr-test.yml#L204

**Review by artursouza — CHANGES_REQUESTED** [review]

**Review by artursouza — APPROVED** [review]

File: .github/workflows/fossa.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v3
+
+      - name: "Run FOSSA Scan"
+        uses: fossas/fossa-action@main # Use a specific version if locking is preferred
+        with:
+          api-key: f886d86744fc734c13de463b87eb3482 # This is a PUSH ONLY token that is safe to be shared with the public
```
Comment (shubham1172) [inline .github/workflows/fossa.yaml:41]: @berndverst we will need to change this with the central token that we get from CNCF once Mark helps us get one.

Comment (berndverst) [reply .github/workflows/fossa.yaml:41]: Yes of course and additional we have to actually set up the different project in that central FOSSA org. But I doubt that will happen that soon.

Comment (shubham1172) [reply .github/workflows/fossa.yaml:41]: I am targeting v1.7. If we don't get a central account by then, I will create workflows across repos with my personal push token.

**Pull request merged**